docs/operations/logging.rst

```diff
@@ -1,7 +1,7 @@
 Configuring logging
 -------------
 
-All logs are sent to ``stdout`` by default. You can change this by `extending the images <development.html/_extending_images>`_. 
+All logs are sent to ``stdout`` by default. You can change this by `extending the images <../development.html#extending_images>`_. 
```

Comment:
need to use RST anchors, e.g.

```
:ref:`extending_images`
```

although you also need to fix this (extra space after anchor and `:`: https://github.com/confluentinc/cp-docker-images/blame/3.1.0-post/docs/development.rst#L170